src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1

```diff
@@ -167,20 +167,36 @@ class SVTResourceResolver: AzSKRoot {
             $this.IsAIEnabled = $true; 
         }
         
-        #Checking if org name is correct 
-        $apiURL = "https://dev.azure.com/{0}/_apis/Contribution/HierarchyQuery?api-version=5.0-preview.1" -f $($this.organizationName);
-        
-        $inputbody = "{'contributionIds':['ms.vss-features.my-organizations-data-provider'],'dataProviderContext':{'properties':{'sourcePage':{'url':'https://dev.azure.com/$($this.organizationName)','routeId':'ms.vss-tfs-web.suite-me-page-route','routeValues':{'view':'projects','controller':'ContributedPage','action':'Execute'}}}}}" | ConvertFrom-Json
-        try {
-            $responseObj = [WebRequestHelper]::InvokePostWebRequest($apiURL, $inputbody);
-            $organizationId = ($responseObj[0].dataProviders."ms.vss-features.my-organizations-data-provider".organizations | Where-Object {$_.name -eq $this.organizationName}).id
-            $inputbody = $null;
-            Remove-Variable inputbody;
+        try { 
+            if (-not [string]::IsNullOrWhiteSpace($env:RefreshToken) -and -not [string]::IsNullOrWhiteSpace($env:ClientSecret))
+            {
+                $apiURL = "https://app.vssps.visualstudio.com/_apis/accounts"
+                $responseObj = "";
+                
+                $responseObj = [WebRequestHelper]::InvokeGetWebRequest($apiURL) ;
```

Comment:
do we realy need this api call to get organizationid? we already called org api in SVTResourceResolver and get orgId, cant we use same one?

Reply:
This is the only api call in entire module where we re getting orgid, correct me if i am wrong

Reply:
ya  its  same place